jenkins/janitor.py

```diff
@@ -96,7 +96,7 @@ def clear_resources(project, col, resource):
     """
     err = 0
     for col, items in col.items():
-        if args.dryrun:
+        if args.dryrun is True:
```

Comment:
I think with `default=False` and `action='store_true'` you can just use implicit truth here.